packit-ci.fmf

```diff
@@ -22,8 +22,8 @@
 
   discover:
     how: fmf
-    url: https://github.com/RedHat-SP-Security/keylime-tests
-    ref: main
+    url: https://github.com/ansasaki/keylime-tests
+    ref: fix_secure_mount
```

Comment:
Is this really required? If necessary ... why not updating the tests in RedHat-SP-Security? Normally, it is not a good practice to use personal forks ...

Reply:
Yes, this is necessary. The changes were proposed in: https://github.com/RedHat-SP-Security/keylime-tests/pull/798

We usually to this to be able to test before the PR is merged on `keylime-tests` side. This is the way we've been testing while we don't have the test merged there, and also to confirm that the fixes to the tests are correct.

Reply:
The test changes were merged in the `keylime-tests` side. I dropped the commit changing the location of the tests.